manual/docs/installation.md

```diff
@@ -0,0 +1,368 @@
+# Clone the repository
```

Comment:
What's the difference between this file and `manual/docs/installation.md` and `INSTALL.md` in the root of the repo? Can we reduce the duplication here?